test/unit/test-SqlString.js

```diff
@@ -257,6 +257,13 @@ test('SqlString.format', {
     assert.equal(sql, 'SELECT * FROM `table` WHERE id = 42');
   },
 
+  'double quest marks passes pre-escaped id': function () {
+    var sql = SqlString.format(
+      'SELECT * FROM ?? WHERE id = ?',
+      [SqlString.identifier('table'), 42]);
```

Comment:
I can't quite put my finger on why, but it seems like if `SqlString.identifier` is an API, this is important for some kind of symmetry.